### PR TITLE
DEV: added Setpoint Hysteresis PV to expert screen. Also modified at_…

### PR DESCRIPTION
**Pull request by slacAdpai**
…vac setpoint to protection setpoint, which is what the underlying code is actually doing. Need to Fix the vacuum library pragma in the next library update.

## Description
Adding Ion Pump setpoint hysterisis to the PLC control. Now ion pumps will interlock correctly base on hysterisis values.

### CODE REVIEW

File: pcdsdevices/pump.py [hunk 1]
```diff
@@ -160,6 +160,10 @@ class PIPPLC(Device):
                           doc='high voltage digital output')
     interlock_ok = Cpt(EpicsSignalRO, ':ILK_OK_RBV', kind='normal',
                        doc='interlock  is ok when true')
+    protection_setpoint = Cpt(EpicsSignalWithRBV, ':AT_VAC_SP', kind='omitted',
+                              doc='Protection/At Vacuum Setpoint')
+    setpoint_hysterisis = Cpt(EpicsSignalWithRBV, ':SP_HYS', kind='omitted',
```
Comment (klauer) [inline pcdsdevices/pump.py:165]:
```suggestion
    setpoint_hysteresis = Cpt(EpicsSignalWithRBV, ':SP_HYS', kind='omitted',
```

File: pcdsdevices/pump.py [hunk 2]
```diff
@@ -160,6 +160,10 @@ class PIPPLC(Device):
...
+                              doc='Protection Setpoint Hysterisis')
```
Comment (klauer) [inline pcdsdevices/pump.py:166]:
```suggestion
                              doc='Protection Setpoint Hysteresis')
```

**Comment by slacAdpai** [comment]
Thank you! I can't spell!

**Comment by klauer** [comment]
Test failures appear unrelated to this PR, but rather bluesky fly-scanning changes... (@ZLLentz ?)

**Comment by ZLLentz** [comment]
The fly scan smoke test for the sequencer failed... and it failed on collect... and collect had a PR go through in bluesky recently. Gotta put on the bugsquashing hat later. This PR did not cause the failure.

**Pull request merged**